Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.yml

```diff
@@ -220,6 +220,26 @@ configuration:
   hidden:
   - marketplacev2
   required: false
+- additionalinfo: 'Define how to close the mirrored incidents from Cortex XSOAR into Defender with a custom close reason mapping. Enter a comma-separated list of close reasons (acceptable format {Cortex XSOAR close reason}={Defender close reason}) to override the default close reason mapping defined by Cortex XSOAR. Note that the mapping must be configured accordingly with the existing close reasons in Cortex XSOAR and Defender. Not following this format will result in closing the incident with a default close reason. Example: "Resolved=TruePositive-Malware,False Positive=FalsePositive-NotMalicious". Refer to ../README.md for possible close-reasons - `Defender Incident Mirroring`.'
+  display: Custom close-reason mapping (XSOAR -> Defender mirrored incident. Overwrites default close-reason mapping defined by Cortex XSOAR)
```

Comment:
Try to short the display etc in order to display proper in the UI
the same in the other parametrs